src/routes/NewRequest/components/NewRequestPage/useNewRequestPage.js

```diff
@@ -107,25 +107,14 @@ export default function useNewRequestPage() {
       requestPublicInfo.createdBy = user.uid;
       requestPrivateInfo.createdBy = user.uid;
 
-      const userRef = firestore.doc(
-        `${USERS_PRIVILEGED_COLLECTION}/${user.uid}`,
-      );
+      const userRef = firestore.doc(`${USERS_COLLECTION}/${user.uid}`);
       const profile = (await userRef.get()).data();
-      let nameParts = user.displayName?.split(' ');
-      if (nameParts.length < 2) {
-        nameParts = [user.displayName, ''];
-      }
-      if (profile) {
-        profile.displayName = user.displayName;
-        [profile.firstName, profile.lastName] = nameParts;
-
-        userInfo = {
-          uid: user.uid,
-          firstName: profile.firstName,
-          displayName: profile.displayName,
-        };
-        requestPublicInfo.createdByInfo = userInfo;
-      }
+      userInfo = {
+        uid: user.uid,
+        firstName: profile.firstName,
+        displayName: profile.displayName,
```

Comment:
Should we do `|| ''` here to handle the case of no display name like you were mentioning happens on email login?